README.md

```diff
@@ -3,7 +3,7 @@
 ## About
 
 This is the home of [Developing Backbone.js Applications](http://shop.oreilly.com/product/0636920025344.do), an open-source book about the Backbone.js library for structuring JavaScript applications. It is released under a 
-Creative Commons Attribution-Noncommercial-No Derivative Works 3.0 United States License and is currently available for early purchase via O'Reilly if you would like to support the project.
+Creative Commons Attribution-Noncommercial-No Derivative Works 3.0 United States License. If you would likt to support the project it is available for early purchase via O'Reilly.
```

Comment:
s/likt/like

Reply:
The changes here mostly seem fine but 👍 to the above - it should say like